src/Guide/Api/Methods.hs

```diff
@@ -238,3 +275,27 @@ dbQuery :: (MonadIO m, EventState event ~ GlobalState, QueryEvent event)
         => DB -> event -> m (EventResult event)
 dbQuery db x = liftIO $
   Acid.query db x
+
+-- | Helper. Get Category from database and throw error when Nothing.
```

Comment:
"Nothing" is an implementation detail, and there's no need to expose it.

Also, when referring to entities or concepts, either use `'Category'` to point to the type, or don't use any Special Capitalization at all.

```suggestion
-- | Helper. Get a category from database and throw error 404 when it doesn't exist.
```